### PR TITLE
librbd: perf section name: use hyphen to separate components

### PR DESCRIPTION
**Pull request by trociny**
"/" and "@" characters make invalid xml format output.

Fixes: #13719
Signed-off-by: Mykola Golub <mgolub@mirantis.com>

### CODE REVIEW

File: qa/workunits/rbd/test_admin_socket.sh [hunk 1]
```diff
@@ -36,9 +36,15 @@ function rbd_get_perfcounter()
 {
     local image=$1
     local counter=$2
+    local name
 
-    ceph --admin-daemon $(rbd_watch_asok ${image}) perf dump |
-	sed -ne 's/^.*"'${counter}'": \([0-9]*\).*$/\1/p'
+    name=$(ceph --format xml --admin-daemon $(rbd_watch_asok ${image}) \
+		perf schema | $XMLSTARLET el -d3 |
```
Comment (jdurgin) [inline qa/workunits/rbd/test_admin_socket.sh]: missing the definition of XMLSTARLET, otherwise looks good

Comment (trociny) [inline qa/workunits/rbd/test_admin_socket.sh]: XMLSTARLET is defined in qa/workunits/ceph-helpers.sh, which is included above.

**Comment by trociny** [comment]
Rebased to kick make check bot.

File: qa/workunits/rbd/test_admin_socket.sh (hunk 1 above)
Comment (jdurgin) [inline qa/workunits/rbd/test_admin_socket.sh]: ah, missed that. will merge in the morning

**Pull request merged**